### DISCUSSION TITLE
Physical location of uploaded files in AtroDAM

### DISCUSSION THREAD

**Discussion by verkstaden-one** [atrocore/atrocore#206] [Q&A] [answered]
I'm brand new to AtroDAM and just got it installed on my server.  I am hoping to utilize this to organize hundreds of gigabytes of my photographs and design files and am curious where the physical files are stored on the server?  I have a RAID5 array attached to the server where all of these files are currently stored.  Is there a way to specify the storage location?  I can't find anything in preferences or administration settings that might indicate it's possible.  Thank you for any help!

Update: I just uploaded some files and found they are stored in an endless tree of directories under "upload" in the root.  I'm going to try and link the upload directory to a folder on my RAID array and see if I can change the storage location that way.  I'll update with results in case anyone else is curious about doing this.

**Comment by atrolex** [comment, marked as answer]
Hi, we will implement the requested functionality within next 2-3 weeks. 
You will be able to put all your digital assets inside of the Upload Directory (it can be configured which directory is your Upload Directory).
Then you can use Sync Function (which will be implemented), which will go through the tree with all you assets, create records in the database and create all thumbnails for them. 
It will be possible to add new files physically (not via Web UI) and use the Sync Function again.
Please wait :)

**Reply by verkstaden-one** [reply]
That sounds fantastic and thank you for the heads up!

**Reply by marcus-at-localhost** [reply]
Does the sync already exists?

**Reply by atrolex** [reply]
No, still on the roadmap, too much with PIM to do.

**Reply by paultvis** [reply]
@atrolex, did this ever get implimented? Would be a useful feature

**Reply by ghost** [reply]
will it be possible to set the upload folder to be on a totally different server as well?

**Reply by atrolex** [reply]
In the future yes, it will be possible to use multiple storages, eg Amazon S3, etc..
We plan to implement some functionality here till the end of Q2 23.
